pkg/build/api/types.go

```diff
@@ -142,6 +146,9 @@ const (
 
 	// BuildPhaseCancelled indicates that a running/pending build was stopped from executing.
 	BuildPhaseCancelled BuildPhase = "Cancelled"
+
+	// BuildPhaseSkipped indicates that a queued build was skipped.
+	BuildPhaseSkipped BuildPhase = "Skipped"
```

Comment:
@smarterclayton are we considering https://github.com/kubernetes/kubernetes/issues/7856 for OS resources?